src/WebJobs.Script/Workers/FunctionInvocationDispatcherFactory.cs

```diff
@@ -30,7 +30,8 @@ internal class FunctionInvocationDispatcherFactory : IFunctionInvocationDispatch
             IWebHostRpcWorkerChannelManager webHostLanguageWorkerChannelManager,
             IJobHostRpcWorkerChannelManager jobHostLanguageWorkerChannelManager,
             IOptions<ManagedDependencyOptions> managedDependencyOptions,
-            IRpcFunctionInvocationDispatcherLoadBalancer functionDispatcherLoadBalancer)
+            IRpcFunctionInvocationDispatcherLoadBalancer functionDispatcherLoadBalancer,
+            IOptions<WorkerConcurrencyOptions> concurrencyOptions)
```

Comment:
nit: here and elsewhere, name the param workerConcurrencyOptions? To distinguish between ConcurrencyOptions, which we also use in places?